substratevm/mx.substratevm/suite.py

```diff
@@ -1992,6 +2010,19 @@
           "testDistribution" : True,
         },
 
+        "SVM_TESTS WITH SPACE" : {
```

Comment:
Might warrant a comment that there is **intentionally** a space in the name. Hard to spot ;-)

Reply:
Yes indeed. @zakkak please add a comment in `suite.py` why we need exactly that.

Reply:
Done.